packages/block-editor/src/store/selectors.js

```diff
@@ -1115,41 +1096,6 @@ const canIncludeBlockTypeInInserter = ( state, blockType, rootClientId ) => {
 	return canInsertBlockTypeUnmemoized( state, blockType.name, rootClientId );
 };
 
-/**
- * Returns whether we can show a reusable block in the inserter
- *
- * @param {Object} state Global State
- * @param {Object} reusableBlock Reusable block object
- * @param {?string} rootClientId Optional root client ID of block list.
- *
- * @return {boolean} Whether the given block type is allowed to be shown in the inserter.
- */
-const canIncludeReusableBlockInInserter = ( state, reusableBlock, rootClientId ) => {
-	if ( ! canInsertBlockTypeUnmemoized( state, 'core/block', rootClientId ) ) {
-		return false;
-	}
-
-	const referencedBlockName = getBlockName( state, reusableBlock.clientId );
```

Comment:
Since a reusable block can have contain blocks, no blocks or a single block, I don't think it makes sense to validate if the content is "includable" in the inserter, we should only validate if `core/block` is allowed.

This was inconsistent before because of the `core/template` trick we had.

Reply:
Out of curiosity, is it something which could be extracted to its own pull request, if you're suggesting it ought not have been implemented this way in the first place?

Reply:
No, I don't think it can because right now the reusable block `edit` function uses this parsed block with `BlockEdit` component to render it and update. It's very much tied to the way the reusable blocks are edited today as part of the global editor.